man1/aur-build.1

```diff
@@ -30,7 +30,8 @@ specified, the current directory is assumed.
 
 .TP
 .B \-c
-Build packages with \fBaur\-chroot\fR(1).
+Build packages with \fBaur\-chroot\fR(1). Also accepts
```

Comment:
"Also accepts" is somewhat vague, perhaps replace it with "The ... options are forwarded to aur-chroot"?